### DISCUSSION TITLE
What was Madonna's full name at 🍼?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#904] [Galaxy Trivia] [answered]
**Bonus:** How many times has she been 👰‍♀️?

**Comment by peterbaccaro** [comment, marked as answer]
Madonna's full name at birth was **Madonna Louise Ciccone**.

Madonna has been married **two** times. 
- Her first marriage was to actor Sean Penn from 1985 to 1989. 
- Her second marriage was to director Guy Ritchie from 2000 to 2008.
